ci-demo/Jenkinsfile

```diff
@@ -27,7 +27,7 @@ pipeline{
             }
         }
         stage('Scan') {
-            steps {        
+            steps {
                 sh "docker run --rm -v /var/run/docker.sock:/var/run/docker.sock sysdig/sysdig-inline-scan -s ${SYSDIG_SECURE_URL} -k ${SYSDIG_SECURE_TOKEN} ${IMAGE_NAME}"
```

Comment:
Maybe this is not intended to be changed in this PR, but:
- We are using here `sysdig-inline-scan` instead of `secure-inline-scan`
- Should we pin this to secure-inline-scan:2?
- Do we still need to mount /var/run/docker.sock in this example

Reply:
In fact, I don't know where this demo is being used, and how it works, because sysdig/sysdig-inline-scan does not exist anymore. If we can find where this demo is being used, I would fix it to use V2. 

If we cannot find it, I would completely remove it, as there are enough Jenkins examples over the docs.

Reply:
I vote for remove. It can be retrieved from git in case we want to.